rpcs3/rpcs3qt/tooltips.h

```diff
@@ -170,7 +170,11 @@ class Tooltips : public QObject
 
 		// network
 
-		const QString net_status = tr("Leave as disconnected unless you're debugging.\nRPCS3 has no online support.");
+		const QString net_status = tr("If set to online, RPCS3 will allow programs to use your internet connection.");
+		const QString psn_status = tr("If set to Fake, RPCS3 will fake PSN connection as best as it can.");
```

Comment:
```suggestion
		const QString psn_status = tr("If set to Simulated, RPCS3 will fake PSN connection as best as it can.");
```
This or option change from Simulated to Fake.